### ISSUE TITLE
"C" as Celsius recognized, but "°C" is not

### ISSUE THREAD

**Issue by g-liu** [gentooboontoo/js-quantities#50]
It would be wonderful if the degree symbol was recognized as part of a temperature unit.

**Comment by MikiDi** [comment]
I agree that support for °C would be great, although it's not easy to implement as the temperature units are a very special case (discussion here #1 and here #37) ... btw, 'C' is the SI-symbol for Coulomb (that's why it was recognized, see [here](https://github.com/gentooboontoo/js-quantities/blob/master/src/quantities.js#L193)), you can use ['tempC'](https://github.com/gentooboontoo/js-quantities/blob/master/src/quantities.js#L148) for a temperature in °C

**Comment by MeirionHughes** [comment]
Is this really that difficult? the definition: "By international agreement, since 1954 the unit "degree Celsius" and the Celsius scale are defined by absolute zero... This definition also precisely relates the Celsius scale to the Kelvin scale, which defines the SI base unit of thermodynamic temperature with symbol K "

**Comment by bcowgill** [comment]
Also one can use the unicode ℃  ℉ as an alias
